### PR TITLE
Goerli Deployment

### PR DESCRIPTION
**Pull request by daryakaviani**
# Task: Goerli deployment

## Description

This PR deploy all Squeeth contract other than the CrabMigration to Goerli, add addresses to the frontend package and deploy subgraph.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

**Comment by haythem96** [comment]
Screen Shot 2022-08-26 at 12 20 21 PM

**Comment by haythem96** [comment]
Still need to deploy Crab V2

**Comment by haythem96** [comment]
Deployed Crab V2

```
reusing "CrabMigration" at 0x5632367328327189A4858ac413A6424fe12F6C8f
reusing "Timelock" at 0x95A9A4A6c9F8Ca5e92a498682a91aa7821C3c0b6
deploying "CrabStrategyV2" (tx: 0x308e010a211132b5e5a6b43e3c56271cff295a5a915e56cf44f77c29e063ae15)..
.: deployed at 0x4Eb533506f996dea20f3d582C2ff36BA527ba35D with 5498385 gas
Successfully deploy CrabStrategyV2
```

File: packages/subgraph/config/ropsten-config.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "UniswapV3Factory-start-block": 11342317,
   "WPowerPerp": "0xa4222f78d23593e82Aa74742d25D06720DCa4ab7",
   "WPowerPerp-start-block": 11808030,
-  "CrabStrategyDeployment": "0xbffBD99cFD9d77c49595dFe8eB531715906ca4Cf",
+  "CrabStrategyDeployment": "0x0637A9184cf4DD294F8AfA3963DeAC2Dfadfb00F",
```
Comment (haythem96) [inline packages/subgraph/config/ropsten-config.json:25]: Not sure why this changed, but can ignore it I guess

Comment (haythem96) [reply packages/subgraph/config/ropsten-config.json:25]: Also should we remove Ropsten from frontend as only Goerli will be maintained?

**Comment by haythem96** [comment]
Subgraph endpoints:
Queries (HTTP):     https://api.thegraph.com/subgraphs/name/opynfinance/squeeth-goerli
Subscriptions (WS): wss://api.thegraph.com/subgraphs/name/opynfinance/squeeth-goerli

File: .circleci/config.yml [hunk 2]
```diff
@@ -22,9 +22,9 @@ workflows:
       - contract-size:
           requires:
             - compile
-      - deploy-test:
-          requires:
-            - compile
+      # - deploy-test:
```
Comment (haythem96) [inline .circleci/config.yml:25]: I commented this, not sure why we deploy using CI?!

Comment (akwritescode) [reply .circleci/config.yml:25]: better to just remove the lines than leave them commented there

**Comment by haythem96** [comment]
New final addresses:

```
Start deploying with 0x59a9303907038A2be0e8837343E1A93Bd69004A3
Using WETH9 at 0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6
Using USDC at 0xd87ba7a50b2e7e660f678a895e4b72e7cb4ccd9c
Start deploying with 0x59a9303907038A2be0e8837343E1A93Bd69004A3
Already have Uniswap Deployment on network goerli. Skipping this step. 🍹

deploying "Oracle" (tx: 0x2c077268e17cf225614e6ef6caff166bbdebd3d7a5acc8633f578485a585f5a4)...: deployed at 0x05b270EC6A386302595A32c16a855a7325f7128D with 874024 gas
deploying "ShortPowerPerp" (tx: 0x3641bdf5e5fc1d8cf588663c01ef1345a4b76b0d39d0b2e09e972bded5ecbf16)...: deployed at 0x426131b367e763f29075a0F064bB502424AA1de1 with 1944244 gas
deploying "WPowerPerp" (tx: 0x914e702ffbe956fedba62a43340f1a810c829d9dad0b564b0684e848fb8be7fa)...: deployed at 0xE9BB90c4f89e895324668658d0208c19c0821A7D with 1032138 gas
SQU/ETH Pool created 🐑. Address: 0x4FbB36C3aA41291b54a5C4cc9AAD1CBF5e1F7cd6
squeethEthPool address 0x4FbB36C3aA41291b54a5C4cc9AAD1CBF5e1F7cd6
ethUSDCPool address 0x04B1560f4F58612a24cF13531F4706c817E8A5Fe
deploying "ABDKMath64x64" (tx: 0xaaf0aa69ef9dd934259de820b3847593a374879b66440eed502b72a06b197f2c)...: deployed at 0x739a1bda66bDCB7567D183AceE4F9cB2696F46a7 with 941465 gas
deploying "TickMathExternal" (tx: 0xd19ea50b1365b64ad4b489da7e3dbd590b2d1adf44c18a3b96ba41954d8fc675)...: deployed at 0x58A2b915C5E5882Af8eD01D4BE0ee4a09C20fd1F with 461403 gas
deploying "SqrtPriceMathPartial" (tx: 0x1c5d6d0744d537ba759a12a2011e662c31a1e7e17e02179b39301c1ff90b0f33)...: deployed at 0x94098668200cec21352CBE76cB6bc672de46c47b with 245472 gas
deploying "Controller" (tx: 0x092724d2a6f796aae4c7c012607dde647ba4782d30df6397287c24bb5aee0055)...: deployed at 0x2CD60Bb7583145BD19560B698b84f23f05E2Bd5e with 5402942 gas
Squeeth init done 🍋
ShortPowerPerp init done 🥭
deploying "ShortHelper" (tx: 0x297f4ebee09ffe7ac2b44f69b2eff8df267b7513e9c05826358333ea0272ad4c)...: deployed at 0xCB2Fad00D6C56Dd937056f76286dd64281F6C347 with 1067567 gas
Successfully deploy ShortHelper 🥦
deploying "CrabStrategyDeployment" (tx: 0xf0ac674ada1772857a4b79e5b8dd58238642c0d6935dd41bdc60ca78ce8f285f)...: deployed at 0x281b14c2fe12411CE18D34d86032dc54E379Ef18 with 5996942 gas
Successfully deploy CrabStrategy
deploying "CrabStrategyV2" (tx: 0x7a948896bb326a26d326564b902758d6f29bc636b74a068256792ea245f2ab42)...: deployed at 0xA6c8445f56dB0Dc7aa8fEE0bd2300dF045E2f78e with 5507838 gas
Successfully deploy CrabStrategyV2
```

**Comment by haythem96** [comment]
```
Start deploying with 0x59a9303907038A2be0e8837343E1A93Bd69004A3
OpynUSDC Deployed at 0x061Ab84aB0fE23C3fD09Cd94820f9592fc983FFB 🍇
OpynWeth Deployed at 0x71Cc058a2338Ad211AC2aFd3fCdb10F2e1bb7741 🍇
```

**Comment by haythem96** [comment]
```
Start deploying with 0x59a9303907038A2be0e8837343E1A93Bd69004A3
Already have Uniswap Deployment on network goerli. Skipping this step. 🍹

deploying "Oracle" (tx: 0x8c8c92d15c184dc2063309e929758cf1204065d39ecd2e63b70fe26b8d616aac)...: deployed at 0xD0cA89609FD8Ca9aa3e683BAf452b3e843157384 with 874024 gas
deploying "ShortPowerPerp" (tx: 0x22d745e5ef2023eb8646926a2a736e613621788c545b7fbd0b1a004d6d34bffc)...: deployed at 0x8af976f16873cC7a9F90c1276371cFA2Ba7B2d82 with 1944244 gas
deploying "WPowerPerp" (tx: 0x5a1b0e8272d0183c2f47766f10cbd0e7d4c8151abcad05b3537e7f0dab8dea7e)...: deployed at 0x38bF9e0444C2a331e34A9c2c72eCAE351977B8d4 with 1032138 gas
SQU/ETH Pool created 🐑. Address: 0x5A44D7455C0F7Ee9A0Bcc6a63719F16c04EbA6f6
ETH/USD Pool created 🐑. Address: 0x9868F3C11bB2996954E700773BF8CdA636801440
deploying "ABDKMath64x64" (tx: 0x47b59a423295704fe2c10d082706d5a72e6af7a7b164b7924e92fb644ea15859)...: deployed at 0x3B364b905F46A912A3DFc134f19B0A0A10f1Ca44 with 941465 gas
deploying "TickMathExternal" (tx: 0x59d4c27dd0a4adc31797232aaffe4fd079f1445578f55c21a65f41e1ad6d1780)...: deployed at 0x1153413A95081449B3e734Ac30c6D5C353d45042 with 461403 gas
deploying "SqrtPriceMathPartial" (tx: 0x88d53a0922b0f6afa8a24ea4b1ea3752b702b61626cd7d3485a85798e5198370)...: deployed at 0xa9a5D75F0bA49f556380645cA2e5Da340C0653B1 with 245472 gas
deploying "Controller" (tx: 0x02cd08d2569c9e685eb360b36a0a87704f18d2c0e03eb6cadd59a12a658c6e9c)...: deployed at 0x1e64d2616EC34aDeD8Bf38394e5e5387Ce13674B with 5402930 gas
Squeeth init done 🍋
ShortPowerPerp init done 🥭
deploying "ShortHelper" (tx: 0x493f4bb9fa33f673137d187dd50566d699e4a77ea67ec5a9927eb3183f1e0b2b)...: deployed at 0x7aB78a716E017eC98f8c9fc549d96097a7A2b866 with 1067674 gas
Successfully deploy ShortHelper 🥦
deploying "CrabStrategyDeployment" (tx: 0xb89404a24a0b66621e89d959203353a112cbfb8201542a36b9d57ecd15c38a15)...An unexpected error occurred:
```

**Comment by haythem96** [comment]
```
deploying "CrabStrategyDeployment" (tx: 0xbf749e71bc2c016f6c7b10f964e69ccef2447f8f4082d64eae47753e4a2f553f)...: deployed at 0x643A36Cd5e50c8e2B4B2B16086FE30c42C017590 with 5962952 gas
Successfully deploy CrabStrategy
deploying "CrabStrategyV2" (tx: 0x62908ed6d54efdb532eefbe5e1b5e9e74bd13a3dbb8216aed7475828fc5df751)...An unexpected error occurred:
```

**Comment by haythem96** [comment]
Start deploying with 0x59a9303907038A2be0e8837343E1A93Bd69004A3
OpynUSDC Deployed at 0x12F263aAB668aF8918E077af3a9CF5da9fE9A417 🍇
OpynWeth Deployed at 0x0719E63EC564259D1ce12dFFD1431269C7d88700 🍇

**Comment by haythem96** [comment]
Deploying whole Uniswap 🦄 on network goerli again...
deploying "UniswapV3Factory" (tx: 0x3d892933b2c41d8829984aed709ef4b526c820a609070eca474331d630f9a600)...: deployed at 0x54e8D71D15052A3f332F5F0E69F6c4456563a93B with 5439096 gas
UniswapV3Factory Deployed 🍹
deploying "SwapRouter" (tx: 0x7423763d79f39b7b7127b87465c07eae0ba65007135b3627cf170f5370f57b43)...: deployed at 0xF9899B5D67410c74de7826781afB09829b23919F with 2667108 gas
SwapRouter Deployed 🍍
deploying "NonfungiblePositionManager" (tx: 0xc4525ea056410373013562de6e8793301f8c049ddbcda140e723c522ea7dc9f8)...: deployed at 0x6cf1eB8e7761c07e4baDfBf5F767D2089ac4cB66 with 5471278 gas
NonfungiblePositionManager Deployed 🥑

deploying "Quoter" (tx: 0xa7171e062642f1cf9a51e08b788e9b8964545a9b4e71e621ce758d907e333886)...: deployed at 0x968Fbd0a650cD6F8D62bf1B39De1b3D69b12af14 with 1836353 gas
Quoter Deployed  🥦

**Comment by haythem96** [comment]
deploying "Oracle" (tx: 0x8809ea743db7ab2e16a32be5d8f133c7ec1303e3b2a4fcda6755ebf3918829d9)...: deployed at 0x7869f5788B54Fa7c46a09C2489a9513609c230a1 with 874024 gas
deploying "ShortPowerPerp" (tx: 0x274c678dbf2f83b262d1377f1588b44fcb50c960164eab1d6a31f53517a1a402)...: deployed at 0x5312077E974bf71302Db7c3a600C4E88B6B4df15 with 1944244 gas
deploying "WPowerPerp" (tx: 0x6556df5d11760f120fae0fca8e66bcd08e24db4f3506bc53332670bef918e6d9)...: deployed at 0xE4E05319dD9fB267C695cFECbAB1c219D155DFc7 with 1032138 gas
SQU/ETH Pool created 🐑. Address: 0x3967eC218425Cf92966b663a2EdE1beED7BAca90
ETH/USD Pool created 🐑. Address: 0xf8F42D258DEfa714DC4FDCAC70a11a51E0Af7f62
deploying "ABDKMath64x64" (tx: 0x33269c2f38dc19d3942b69538d508c0b9b9841588ee6c36d1ac2ed51adc53b08)...: deployed at 0x3831d92475Ed4Ad0B615DefB25b6FBf9B7944D14 with 941465 gas
deploying "TickMathExternal" (tx: 0x12a06a46da4c7b235c5be9255cb527c68f3479b48149eee7e02d99d5ebd5aea6)...: deployed at 0x7D019D9DCc2B5d3AeDBfD929f2AE4c692Df11FDa with 461403 gas
deploying "SqrtPriceMathPartial" (tx: 0xd7849d3400aab621bce75330a3f628fe6ffac5f1d857706c4589f665a9c39b5b)...: deployed at 0x5eB305560a660326E4D419e466AD66F926a38E81 with 245472 gas
deploying "Controller" (tx: 0x51d1288e0303dbcd8d0e1f42b34e12c8e3d2fccef6f6b5ded7d10ef86edb5be6)...: deployed at 0x863a69598637D67AD21d5F15E5c051897b9FeFF8 with 5402930 gas
Squeeth init done 🍋
ShortPowerPerp init done 🥭
deploying "ShortHelper" (tx: 0x668cccd9bdb378e9c14577765de8e1ee48e828b9cc65298343005d0c89b14ebc)...: deployed at 0x75aff4de4fe9444bF18F30CEEB5E1A11a3FFc7d9 with 1067523 gas
Successfully deploy ShortHelper 🥦

**Comment by haythem96** [comment]
deploying "CrabStrategyV2" (tx: 0x99403d5bd0d066ff8de19e3208b51345e5f23780418b56caa8edd1bff3364809)...: deployed at 0x45b08e286f469D1067c8cb2636dEc3e2f5A42cC9 with 5464376 gas
Successfully deploy CrabStrategyV2

**Comment by haythem96** [comment]
deploying "CrabStrategyDeployment" (tx: 0x95d72d86b6b88805ba4990cdeec8add4b09a205a179f97e5c9c75b4139c16bbd)...: deployed at 0xd9313b9C09D5Df0b0E8a00AB0915a30292c7AeE1 with 5945949 gas
Successfully deploy CrabStrategy

**Comment by haythem96** [comment]
Deploying whole Uniswap 🦄 on network goerli again...
deploying "UniswapV3Factory" (tx: 0x795dee37b79b1830e4c3ad3d41fc0e5a31a4621577d1e760237b7f724f8120c9)...: deployed at 0x2Db05f50645a7EAEd686bdf64DE3F90Fde2041b3 with 5439096 gas
UniswapV3Factory Deployed 🍹
deploying "SwapRouter" (tx: 0x5f280b337cd9b29f882328eb1f6099c2d880064fd84636db4558d75edc4ba490)...: deployed at 0x200775Be10EA5dCee478C193E6B6E66E13d878C0 with 2667108 gas
SwapRouter Deployed 🍍
deploying "NonfungiblePositionManager" (tx: 0x97d6d07cab96bd03ab89e0fca62857b434736bb3a03137759d8aaecc50473cef)...: deployed at 0xa22acAd8494B641D374f510CD88DeacABe24B0F9 with 5471278 gas
NonfungiblePositionManager Deployed 🥑

deploying "Quoter" (tx: 0x7a9336925ca20e1d56c00bf9615ec847343da5057086e2a3a5ecdffe8eaaa304)...: deployed at 0x5D9D69aF49130fEb2511fdFc1A55B5746897BdE4 with 1836353 gas
Quoter Deployed  🥦

deploying "Oracle" (tx: 0x4a925821b2848cb0aaa2865a1ac07fae755ba9db680ea37251e29eeb46dad818)...: deployed at 0x3A1B8e5D405F4080442C878aA114B17a1D60E14B with 874024 gas
deploying "ShortPowerPerp" (tx: 0x7d66585d5766aea2167fac3d629f59d41195dd15e99ef2f006f255b0ed8a848b)...: deployed at 0x66a6a5bCF0cc97b317779152A8a8765FE3802E7C with 1944244 gas
deploying "WPowerPerp" (tx: 0x8cf374ea02a0374aac7b206bcbbea8bb81cbdbfa1ed8911bfa983151e08d69da)...: deployed at 0x6b03eD2C590A301E79E2DCe4ce38D7402dC6735a with 1032138 gas

**Comment by haythem96** [comment]
SQU/ETH Pool created 🐑. Address: 0xB583EA07699c5DEa4da084056273d6D70c2C3309
ETH/USD Pool created 🐑. Address: 0x8875e9c9EB0909da889CB3Dc9c5E8856093CE6b0
deploying "ABDKMath64x64" (tx: 0x1fecce38416637f5d5932a40586330418aeb7143a36a9e5cd23979189257bfd4)...: deployed at 0xA469D78845Ef38846B2bf86de358Dc093FF07BC3 with 941465 gas
deploying "TickMathExternal" (tx: 0x5874dfe82b29011d0cf1cbfcd66501899ab2f938f517acfb27d8fd1ce8fef46c)...: deployed at 0x1825A21FaedfDF9174b8f324FA5Bf3ab011AD27d with 461403 gas
deploying "SqrtPriceMathPartial" (tx: 0x80ed4c30eac34af8aeda92499ecdbe6aa908c5df3d4af70ce738370bb4ed6375)...: deployed at 0xac30D9818c9954f665299b2c1f1269CD3F941A5c with 245472 gas
deploying "Controller" (tx: 0x8a2e278475ff40f3a60c217ee08079fbf807fad4c7adc8d7a357df7c5dead685)...: deployed at 0x2c60f986260c8412De7fA3384C7f0Bab1a4F72bf with 5402930 gas
Squeeth init done 🍋
ShortPowerPerp init done 🥭
deploying "ShortHelper" (tx: 0x5a4a5bccf1f1f91ef7030f71835ce8174f5cb4ac8807e4a4d5c886ff2bbfc7a8)...: deployed at 0x03F4a45341B46eDEb7542890e4bEa94868CB679e with 1067523 gas
Successfully deploy ShortHelper 🥦

**Comment by haythem96** [comment]
deploying "CrabStrategyDeployment" (tx: 0xc91b2f8a2a3492a8722f00ff15f91ba9953db1d31aef6406f1dde5d68933ff5d)...: deployed at 0x74b9e08b801b0df247A14c0f91E7143ea95C946B with 5962438 gas
Successfully deploy CrabStrategy

**Comment by haythem96** [comment]
deploying "CrabStrategyV2" (tx: 0xbfa51c820ad4f526d301525b5f145e03797fb9b6528d12c00c5d5185e11f994a)...: deployed at 0xe2AAC9F9F6f49bcE0aF6AeDD90C384c931e09416 with 5447781 gas
Successfully deploy CrabStrategyV2

**Comment by aleone** [comment]
once more unto the breach...

deploying "CrabStrategyV2" (tx: 0x4d4869427232010f2581012b2750becdcad7ac0e524bf152ce298d00f3a97437)...: deployed at 0x49a423Bc9F1dEe5A052b1E72a1dbE9de488d4411 with 5453282 gas
Successfully deploy CrabStrategyV2

**Pull request merged**